internal/server/root_certificate.go

```diff
@@ -12,8 +12,8 @@ import (
 )
 
 // The CertificateAuthority id will always be set to "roots".
-// The const ca_id contains this value
-const ca_id = "roots"
+// The const CA_ID contains this value
+const CA_ID = "roots"
```

Comment:
Curious about this change. Our normal style would actually be `caId` so to me this reads as a change from one form that isn't our normal form to a different form that isn't our normal form :-)

Reply:
I'll update it to CaId- I needed the const to be exported